packages/utils/src/get-top-level-domain.ts

```diff
@@ -0,0 +1,11 @@
+export const getTopLevelDomain =
+  typeof window === 'undefined'
+    ? () => 'localhost'
+    : () => {
+        if (document.location.host.indexOf('localhost') === 0) {
```

Comment:
It is a bit incorrect:

`'localhost.domain.com'.indexOf('localhost') === 0`

Reply:
outdated